client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -143,7 +148,8 @@ class LifecycleManager(appId: String, val conf: CelebornConf) extends RpcEndpoin
       currentShuffleFileCount: LongAdder,
       commitPartitionRequests: util.Set[CommitPartitionRequest],
       handledCommitPartitionRequests: util.Set[PartitionLocation],
-      inFlightCommitRequest: AtomicInteger)
+      inFlightCommitRequest: AtomicInteger,
+      inFlightPartitionCommitRequest: ConcurrentHashMap[Int, AtomicInteger])
```

Comment:
It's a bit confusing about the difference between inFlightPartitionCommitRequest and inFlightCommitRequest

Reply:
rename these to allInFlightPartitionCommitRequest and partitionInFlightCommitRequest instead.